terraform/terraform.tfvars.json

```diff
@@ -0,0 +1,73 @@
+{
+  "accounts": {
+    "production": "933639921819",
+    "preproduction": "540070264006",
+    "development": "705467933182",
+    "management": "311462405659"
+  },
+  "prefixes": {
+    "production": "serve",
+    "preproduction": "preproduction.serve",
+    "development": "development.serve"
+  },
+  "bucket_names": {
+    "production": "serve-opg.opg.digital",
+    "preproduction": "preproduction.serve-opg.opg.digital",
+    "development": "development.serve-opg.opg.digital"
+  },
+  "behat_controller": {
+    "production": 0,
+    "preproduction": 1,
+    "development": 1
+  },
+  "sirius_api": {
+    "development": "https://frontend-integration.dev.sirius.opg.digital/",
```

Comment:
Sirius urls are due to change to `opg.service.justice.gov.uk` ones soon, so worth updating.

Reply:
I've added a ticket to change this, [](https://opgtransform.atlassian.net/browse/SER-173)